static/templates/settings/notification-settings.handlebars

```diff
@@ -72,16 +82,28 @@
               "is_checked" page_params.enable_offline_email_notifications
               "label" settings_label.enable_offline_email_notifications}}
 
-            {{partial "settings_checkbox"
-              "setting_name" "enable_offline_push_notifications"
-              "is_checked" page_params.enable_offline_push_notifications
-              "label" settings_label.enable_offline_push_notifications}}
-
-            <div class="input-group disableable {{#unless page_params.enable_offline_push_notifications}}control-label-disabled{{/unless}}">
+            {{!-- Mobile notifications when offline --}}
+            <div class="input-group {{#unless page_params.realm_push_notifications_enabled}}control-label-disabled{{/unless}}" id="push-warning">
+                <label class="checkbox">
+                    <input type="checkbox" name="enable_offline_push_notifications" id="enable_offline_push_notifications"
+                      {{#unless page_params.realm_push_notifications_enabled}}disabled="disabled"{{/unless}}
+                      {{#if page_params.realm_push_notifications_enabled}}
+                      checked="checked"
+                      {{/if}} />
+                    <span></span>
+                </label>
+                <label for="enable_offline_push_notifications" id="enable_offline_push_notifications_label" class="inline-block">
+                    {{settings_label.enable_offline_push_notifications}}
+                </label>
+                <i class="fa fa-question-circle change_email_tooltip settings-info-icon" {{#if page_params.realm_push_notifications_enabled}}style="display:none"{{/if}}  data-toggle="tooltip"
+                  title="{{t 'The server is not configured for push notifications' }}"></i>
+            </div>
+            {{!-- Mobile notifications when online --}}
+            <div class="input-group disableable {{#unless page_params.realm_push_notifications_enabled}}control-label-disabled{{/unless}}">
                 <label class="checkbox">
                     <input type="checkbox" name="enable_online_push_notifications" id="enable_online_push_notifications"
-                      {{#unless page_params.enable_offline_push_notifications}}disabled="disabled"{{/unless}}
-                      {{#if page_params.enable_online_push_notifications}}
+                      {{#unless page_params.realm_push_notifications_enabled}}disabled="disabled"{{/unless}}
+                      {{#if page_params.realm_push_notifications_enabled}}
```

Comment:
Can you explain this change?

Reply:
When using ```{{#unless page_params.realm_push_notifications_enabled}}disabled="disabled"{{/unless}}```, it checks if the server is configured for push notifications or not. In case it isn't, then the checkbox would be disabled.

Second, ```{{#if page_params.realm_push_notifications_enabled}}
                      checked="checked"
                      {{/if}```, if the checkbox is disabled, then it doesn't check the box.

![screenshot 2018-09-01 at 1 54 13 pm](https://user-images.githubusercontent.com/22795943/44944106-b8db4680-adee-11e8-8dc1-51161efe66df.png)

In the above image, if using ```page_params.enable_offline_push_notifications``` then even when server is not setup and checkbox is disabled, it checked(but disabled)